src/group_theory/monoid_localization.lean

```diff
@@ -1315,3 +1320,97 @@ mul_equiv_of_quotient f
 
 end away
 end localization
+
+end comm_monoid
```

Comment:
This file is getting really long. Do you think there is a natural way to split it into 2 or 3 pieces?

Reply:
Yes, I think it's better to split into the following 3 pieces: the characteristic predicate (`localization_map`), the quotient construction (`localization`), and the localizations away from an element.

Reply:
I made an attempt of splitting. #13578